spec/loadbalancer/README.md

```diff
@@ -23,9 +23,52 @@ docker run -p 8080:80 --env-file $(pwd)/machine.env enterpriseconnect/loadbalanc
 ```
 
```

Comment:
Please provide the following-
1) The use case Spec/requirement.
2) Test script, steps, etc.

@ramaraosrikakulapu @palokam

Reply:
> > The diagram lb-usecase.png does not depict the solution in detail. please purpose a clearer diagram @palokam @ramaraosrikakulapu
> 
> Can you please be more specific?

Sure. in the proposed diagram, 1) what type of traffic to be load-balanced, 2) what is the EC Server/Client usage/environment? 3) what is Nginx to do with EC LoadBalancer connectivity? 4) what does the word "EC" in the cloud try to symbolise? etc.

Reply:
> > > The diagram lb-usecase.png does not depict the solution in detail. please purpose a clearer diagram @palokam @ramaraosrikakulapu
> > 
> > 
> > Can you please be more specific?
> 
> Sure. in the proposed diagram, 1) what type of traffic to be load-balanced, 2) what is the EC Server/Client usage/environment? 3) what is Nginx to do with EC LoadBalancer connectivity? 4) what does the word "EC" in the cloud try to symbolise? etc.

Updated with new diagram as discussed for Phase 1. I will update the diagram once we ready with Phase 2.

Reply:
> 3) what is Nginx to do with EC LoadBalancer connectivity

Please replace In the diagram ```doc/lb-usecase.png``` with ```doc/lb-model.png```, or make improvement based on the ```doc/lb-model.png``` (find in lucidchart)  the proposed diagram does not seem too appeal nor indifference. @ramaraosrikakulapu

Reply:
Agree. We are trying to implement the same as described in ```docs/lb-model.png```. So I preferred not to have diagram for use case.

Reply:
can you move the figure ```doc/lb-model.png``` to this use case section instead?